recipes/mss/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "1.2.0" %}
+
+package:
+  name: mss
+  version:  {{ version }}
+
+source:
+    fn: mss-{{ version }}.tar.gz
+    url: https://bitbucket.org/wxmetvis/mss/get/{{ version }}.tar.gz
+    sha256: 6e15a2a7c65ce676169551660e51ced5b0718ef83c4f93104a0f6b70ed502183
+
+build:
+    number: 0
+    script: python setup.py install
+    entry_points:
```

Comment:
If `Mission Support System` is Python 2 only you have to add `skip: True  # [py3k]` here in the build section.